Core/UserAgentManager.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  UserAgentConfiguration.swift
+//  Core
+//
+//  Created by DuckDuckGo on 27/05/2020.
+//  Copyright © 2020 DuckDuckGo. All rights reserved.
+//
+
+import Foundation
+import WebKit
+
+public class UserAgentManager {
+    
+    public static let shared = UserAgentManager()
+    
+    private var defaultAgentRetreived = false
+    private var userAgent: UserAgent
+    
+    init() {
+        let webview = WKWebView()
+        webview.load(URLRequest(url: URL(string: "https://duckduckgo.com")!))
```

Comment:
Did you try about:blank? Happy to leave it as it is if it's needed.

Reply:
I like that better, thanks!